test/pageObjects/wizardPages/Conditionals/conditionals_CreateCase_ConditionalPage1.js

```diff
@@ -1,37 +1,54 @@
-let CCDStringField = require('../ccd-components/fields/ccdStringField.js');
+let CCDStringField = require('../../ccd-components/fields/ccdStringField.js');
 
 class ConditionalsCreateCasePage1WizardPage extends CreateCaseWizardPage {
 
   constructor() {
     super();
     this._fieldUsedInShowCondition = ''; // TextField | TextFieldOptional
-    this.mandatoryTextField = new CCDStringField('ccd-write-text-field', 'text', 'TextField');
-    this.optionalTextField = new CCDStringField('ccd-write-text-field', 'text', 'TextFieldOptional');
+    this.TextField = new CCDStringField('#TextField');
```

Comment:
variable names should not start with a capital letter

Reply:
done